src/releng_docs/deploy/regular.rst

```diff
@@ -44,6 +86,8 @@ Protocal that we follow is:
    - announce in ``#ci`` channel that a push to production is about to
```

Comment:
Nit: Add "On the **release day** (Thursday)" before "announce that a new deployment to production is going to happen".